### PR TITLE
Kickoff api

### PR DESCRIPTION
**Pull request by agustinbajo**
## **Initial Kickoff**
Data Base and user created successfully 

## **Trello Card** 
https://trello.com/c/wcgB4cfT

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+#test:
+#  <<: *default
+#  database: db/test.sqlite3
+
+#production:
+#  <<: *default
+#database: db/production.sqlite3
```
Comment (leoncalermo) [inline config/database.yml]: remove the commented code :eyes: .

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: localhost
+  username: wbooks-api
+  password: wbooks-api
+
+development:
+  <<: *default
+# database: nombre-de-tu-app_development
```
Comment (leoncalermo) [inline config/database.yml]: remove this commented code too :eyes: .

**Review by leoncalermo — CHANGES_REQUESTED** [review]
don't forget to delete commented code.

**Comment by agustinbajo** [comment]
## **commit 0a42f53572f8fa6a880e002af13df993fe50e7bf**
- RuboCop Gem was added 
- All coments from database.yml were removed

**Review by leoncalermo — APPROVED** [review]
all-good All good

**Review by leoncalermo — APPROVED** [review]
all good :ok_hand:

**Pull request merged**